execution/native/precompiles_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Monax Industries Limited
+// SPDX-License-Identifier: Apache-2.0
+
+package native
+
+import (
+	"testing"
+
+	"github.com/hyperledger/burrow/acm/acmstate"
+	"github.com/hyperledger/burrow/execution/engine"
+	"github.com/hyperledger/burrow/execution/exec"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hyperledger/burrow/acm"
+	"github.com/hyperledger/burrow/crypto"
+
+	"github.com/hyperledger/burrow/execution/evm/asm/bc"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSNativeContractDescription1a(t *testing.T) {
+	signatory := []byte("4fbdd7d412e5dad21d0c4add8b3cdc70ab65a5b3")
+	signature := []byte("e8cd78a90152a1549cfea3dcc8556e25618c67085c39e0b95e3d1ec6dbe9339525e7c61e005954fa67ce0c102cee434d1b111c4b0c52bad0fdc2663ca09451ac1b")
+	messageDigest := []byte("21400fb863b0ee7392af02491b6ca5fb87f68ad6af4867b104282dd8fe8b52e4")
```

Comment:
Could we have a comment describing how these were generated - even better if we can generate them - we have little embedding utiltity that will compile solidity files and put their bytes and metadata into `.sol.go` files, see: https://github.com/monax/platform/blob/master/Makefile#L297-L299